wsmaster/wsmaster-local/src/main/java/org/eclipse/che/api/local/LocalSnapshotDaoImpl.java

```diff
@@ -121,6 +122,13 @@ public synchronized void removeSnapshot(String snapshotId) throws NotFoundExcept
     }
 
     @Override
+    public SnapshotImpl updateSnapshot(SnapshotImpl update) throws NotFoundException, ConflictException, SnapshotException {
+        requireNonNull(update, "Required non-null snapshot");
```

Comment:
Please throw unsupported operation exception instead, this dao is not used. We're going to get rid of it.